src/resources/locales/en.json

```diff
@@ -1,6 +1,7 @@
 {
     "accessibility": {
         "closeDialog": "Close",
+        "bookMenu": "Menu",
```

Comment:
Could you please run the command `npm run i18n-sort` followed by `npm run i18n-typed` so that the locale JSON is in "canonical" form (i.e. all language translations sorted by key alphabetical order, same indentation + trailing line break, etc.)
PS: you only need `npm run i18n-scan` if you need to check that the `i18next` keys used in the source code are all present in the JSONs, and that there are no superfluous / unused keys.
You only need `npm run i18n-check` if you need to compare the "primary" english locale with the "secondary" ones (missing keys are automatically added with an empty string value, redundant keys are removed).

Reply:
It's fixed